### PR TITLE
[SPARK-7351][Streaming][Docs] Add spark.streaming.ui.retainedBatches to docs

### PR DESCRIPTION
**Pull request by zsxwing**
The default value will be changed to `1000` in #5533. So here I just used `1000`.

### CODE REVIEW

**Comment by zsxwing** [comment]
retest this please.

File: docs/configuration.md [hunk 1]
```diff
@@ -1474,6 +1474,13 @@ Apart from these, the following properties are also available, and may be useful
     the new Kafka direct stream API.
   </td>
 </tr>
+<tr>
+  <td><code>spark.streaming.ui.retainedBatches</code></td>
+  <td>1000</td>
```
Comment (srowen) [inline docs/configuration.md]: I think the only question is whether this was intended to be exposed to users or not. What's your argument for that? This should be 100, and changed to 1000 in your other PR if this goes in.

Comment (zsxwing) [inline docs/configuration.md]: This is just like `spark.ui.retainedJobs` and `spark.ui.retainedStages` and should be exposed. I think we can merge this after #5533.

**Comment by zsxwing** [comment]
#5533 is merged. So this one is ready to go.

**Comment by tdas** [comment]
LGTM. Merging.

**Pull request closed without merging**